SportHub/src/modules/SearchDropdownList/components/SearchDropdownList.js

```diff
@@ -34,6 +34,11 @@ const SearchDropdownList = ({ setIsContentSearch, setContentSearchValue }) => {
     }
   }
 
+  const handleRedirectToArticle = async (articleId) => {
+    // using navigate breaks search dropdown (it isn't work until refreshing page)
+    window.location.href = ROUTES.ARTICLE.replace(':articleId', articleId)
```

Comment:
[CONSIDER] why you don't use navigate with passing a parameter articleId to it like in SearchResultsList.js ?